lib/screens/v2/widget_instance/subway_status.ex

```diff
@@ -121,6 +232,17 @@ defmodule Screens.V2.WidgetInstance.SubwayStatus do
     end
   end
 
+  defp serialize_gl_pill_with_branches(route_ids) do
+    branches =
+      route_ids
+      |> Enum.filter(&String.contains?(&1, "Green-"))
+      |> Enum.map(fn "Green-" <> branch ->
+        branch |> String.downcase() |> String.to_existing_atom()
+      end)
```

Comment:
Could `route_ids` ever have duplicates? Wondering if this needs a `|> Enum.uniq()`.

Reply:
Not at this point. We use `alert_routes/1` to get the `route_ids` and there is an `Enum.uniq()` call there.